cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -159,13 +163,26 @@ class LeadImpl extends ServerImpl with Lead
     val storeProperties = ServiceUtils.getStoreProperties(bootProperties.stringPropertyNames()
         .iterator().asScala.map(k => k -> bootProperties.getProperty(k)).toSeq)
 
+    val productName = {
+      GemFireCacheImpl.setGFXDSystem(true)
+      GemFireVersion.getInstance(classOf[SnappyDataVersion],
+        "io/snappydata/SnappyDataVersion.properties")
+      GemFireVersion.createVersionFile()
+      GemFireVersion.getInstance(classOf[GemFireXDVersion], SharedUtils.GFXD_VERSION_PROPERTIES)
+      if (GemFireVersion.isEnterpriseEdition) {
+        "TIBCO ComputeDB"
+      } else {
+        "SnappyData"
+      }
+    }
+
```

Comment:
We can use `SnappySession.isEnterpriseEdition` here.

Reply:
Done